test/src/commands/create/create_test.dart

```diff
@@ -420,6 +425,7 @@ void main() {
               vars: <String, dynamic>{
                 'project_name': 'my_app',
                 'description': '',
+                'executable_name': 'my_app',
```

Comment:
Is not required for this tests right?

Reply:
It is, otherwise, the verification doesn't match.